docs/List.md

```diff
@@ -683,6 +685,49 @@ const Aside = ({ data, ids }) => (
 ```
 {% endraw %}
 
+### Empty page
+
+When there is no result, and there is no active filter, and the resource has a create page, react-admin displays a special page inviting the user to create the first record.
+
+You can use the `empty` prop to replace that page by a custom component:
+
+{% raw %}
+```jsx
+import Button from '@material-ui/core/Button';
+import { CreateButton, List } from 'react-admin';
+
+const Empty = ({ basePath, resource }) => (
+    <div style={{ textAlign: 'center', margin: '1em' }}>
```

Comment:
We try and avoid inline styles in the documentation. Try using `<Box>`, or just remove the inline style for the example.